source/resampling_with_code2.Rmd

```diff
@@ -610,26 +654,24 @@ z
 
 Finally, we need to count how many of the trials in `z` ended up with all-White juries.  These are the trials with a `z` value of 0.
 
-We can ask this question of *all 100 counts* by asking the question: is the {{< var array_name >}} `z` equal to 0, like this:
+We can ask this question of *all 100 counts* by asking the question: is the {{< var array >}} `z` equal to 0, like this:
 
 ```{python, opts.label="py_ed"}
 # Is the value of z equal to 0?
-all_white = z == 0
+all_white = (z == 0)
```

Comment:
Do we want a previous example on the lines of:

```
>>> range(10)
range(0, 10)
>>> one_z = 3
>>> one_z == 0
False
>>> two_z = np.array([3, 0])
>>> two_z == 0
array([False,  True])
```

Reply:
Added an example.